### DISCUSSION TITLE
Integrating HTML files?

### DISCUSSION THREAD

**Discussion by spacemishka** [getnikola/nikola#3796] [Q&A]
Hi,

i am using CherryTree and export Content as HTML. I would like to have that as well in Nikola. When i just put it in the structure, Nikola completly mess up everything. I guess it takes it as a markdown file. How can i do this?

thanks

**Comment by ralsina** [comment]
In theory, Nikola has support for html input files. You need to create them with html extension and configure a html compiler in your config.

I can provide more details later today
